plexupdate.sh

```diff
@@ -307,6 +307,20 @@ if [ -z "${DISTRO_INSTALL}" ]; then
 			else
 				DISTRO_INSTALL="${REDHAT_INSTALL}"
 			fi
+		elif [ -f /etc/synoinfo.conf ]; then
+		        DISTRO="synology"
```

Comment:
I noticed when I tried to copy this over the version from head. Some of the lines in this change were intended using both tabs and spaces.

Reply:
i changed the spaces to tabs